src/org/robockets/steamworks/Robot.java

```diff
@@ -153,9 +153,21 @@ public void robotInit() {
 		RobotMap.rollerEncoderCounter.setDistancePerPulse(1.0);
 		
 		RobotMap.shooterRollerSpeedController.setInverted(true);
+		RobotMap.climberSpeedController.setInverted(true);
+		RobotMap.climberSpeedController2.setInverted(true); // 55left 75 right
 
-		RobotMap.leftEncoder.setDistancePerPulse(0.26592797783933518005540166204986);
-		RobotMap.rightEncoder.setDistancePerPulse(0.04164859002169197396963123644252);
+		RobotMap.leftEncoder.setDistancePerPulse(0.03435326842837273991655076495132);
```

Comment:
Could be a constant with a value of`4 * Math.PI / 360`. 0.34... is a magic number.